elfi/visualization/visualization.py

```diff
@@ -229,3 +230,42 @@ def plot_traces(result, selector=None, axes=None, **kwargs):
         axes[-1, ii].set_xlabel('Iterations in Chain {}'.format(ii))
 
     return axes
+
+
+def plot_diff(estimated, reference, spec, method='logpdf'):
```

Comment:
The objects are arbitrary as long as they have implemented the same method which is compared.